dist/stacktrace.min.js

```diff
@@ -1,2 +1,2 @@
eof i&&i)if(y.call(i)==N)for(n=i.length;n--;)B(i,n,r);else m(i,function(e){B(i,e,r)});return r.call(e,t,i)};n.parse=function(e,t){var r,n;return $=0,G=""+e,r=D(q()),"$"!=q()&&J(),$=G=null,t&&y.call(t)==w?Z((n={},n[""]=r,n),"",t):r}}}return n.runInContext=e,n}var t="function"==typeof define&&define.amd,r={"function":!0,object:!0},n=r[typeof exports]&&exports&&!exports.nodeType&&exports,i=r[typeof window]&&window||this,o=n&&r[typeof module]&&module&&!module.nodeType&&"object"==typeof global&&global;if(!o||o.global!==o&&o.window!==o&&o.self!==o||(i=o),n&&!t)e(i,n);else{var a=i.JSON,s=i.JSON3,u=!1,c=e(i,i.JSON3={noConflict:function(){return u||(u=!0,i.JSON=a,i.JSON3=s,a=s=null),c}});i.JSON={parse:c.parse,stringify:c.stringify}}t&&define(function(){return c})}.call(this),function(e,t){"use strict";"function"==typeof define&&define.amd?define("stacktrace",["error-stack-parser","stack-generator","stacktrace-gps"],t):"object"==typeof exports?module.exports=t(require("error-stack-parser"),require("stack-generator"),require("stacktrace-gps")):e.StackTrace=t(e.ErrorStackParser,e.StackGenerator,e.StackTraceGPS)}(this,function(e,t,r){function n(e,t){var r={};return[e,t].forEach(function(e){for(var t in e)e.hasOwnProperty(t)&&(r[t]=e[t]);return r}),r}function i(e){return e.stack||e["opera#sourceloc"]}var o={filter:function(e){return-1===(e.functionName||"").indexOf("StackTrace$$")&&-1===(e.functionName||"").indexOf("ErrorStackParser$$")&&-1===(e.functionName||"").indexOf("StackTraceGPS$$")&&-1===(e.functionName||"").indexOf("StackGenerator$$")}};return{get:function(e){try{throw new Error}catch(t){return i(t)?this.fromError(t,e):this.generateArtificially(e)}},fromError:function(t,i){return i=n(o,i),new Promise(function(n){var o=e.parse(t);"function"==typeof i.filter&&(o=o.filter(i.filter)),n(Promise.all(o.map(function(e){return new Promise(function(t){function n(r){t(e)}new r(i).pinpoint(e).then(t,n)["catch"](n)})})))}.bind(this))},generateArtificially:function(e){e=n(o,e);var r=t.backtrace(e);return"function"==typeof e.filter&&(r=r.filter(e.filter)),Promise.resolve(r)},instrument:function(e,t,r,n){if("function"!=typeof e)throw new Error("Cannot instrument non-function object");if("function"==typeof e.__stacktraceOriginalFn)return e;var o=function(){try{this.get().then(t,r)["catch"](r),e.apply(n||this,arguments)}catch(o){throw i(o)&&this.fromError(o).then(t,r)["catch"](r),o}}.bind(this);return o.__stacktraceOriginalFn=e,o},deinstrument:function(e){if("function"!=typeof e)throw new Error("Cannot de-instrument non-function object");return"function"==typeof e.__stacktraceOriginalFn?e.__stacktraceOriginalFn:e},report:function(e,t){return new Promise(function(r,n){var i=new XMLHttpRequest;i.onerror=n,i.onreadystatechange=function(){4===i.readyState&&(i.status>=200&&i.status<400?r(i.responseText):n(new Error("POST to "+t+" failed with status: "+i.status)))},i.open("post",t),i.setRequestHeader("Content-Type","application/json"),i.send(JSON.stringify({stack:e}))})}}});
```

Comment:
Line is too long.<br>Missing semicolon.<br>Expected an identifier and instead saw '=='.<br>Expected an operator and instead saw 'typeof'.<br>Expected an assignment or function call and instead saw an expression.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.<br>'e' is already defined.<br>'t' is already defined.<br>Possible strict violation.<br>Expected '{' and instead saw 'throw'.<br>Expected '{' and instead saw 'return'.<br>'i' is already defined.<br>Use '!==' to compare with 'null'.<br>Too many errors. (50% scanned).